src/internal.c

```diff
@@ -28823,20 +28948,48 @@ int SendClientKeyExchange(WOLFSSL* ssl)
                 case psk_kea:
                 {
                     byte* pms = ssl->arrays->preMasterSecret;
+                #ifdef WOLFSSL_MAXQ10XX_TLS
+                    byte dummy_psk[MAX_PSK_KEY_LEN];
+                    word32 ret_cb;
+                    if (ssl->options.side == WOLFSSL_CLIENT_END) {
+                        // get client_identity
+                        ret_cb = ssl->options.client_psk_cb(ssl,
+                                     ssl->arrays->server_hint,
+                                     ssl->arrays->client_identity,
+                                     MAX_PSK_ID_LEN, dummy_psk,
+                                     MAX_PSK_KEY_LEN);
+                        if (ret_cb == 0 || ret_cb > MAX_PSK_KEY_LEN) {
+                            ERROR_OUT(PSK_KEY_ERROR, exit_scke);
+                        }
+
+                        // save client_identity
```

Comment:
C style `/* */` only please.

Reply:
All instances of // added by this PR have been changed to `/*  */`